### ISSUE TITLE
accessibility turn off automatically

### ISSUE THREAD

**Issue by pharidali** [Gan4x4/sms_locator#1]
Hi !

Thanks for sharing :)

The app doesnot start on boot Android 7.0 7.1 and 8.0

and the accessibility turn off automatically

I hope you will fix those issues 

Thank you

**Comment by pharidali** [comment]
not on gionee, Xiaomi, Lenovo on boot

**Comment by pharidali** [comment]
I have got a solution for boot problem
https://stackoverflow.com/a/41562672/3264501

It's working now :+1:

**Comment by Gan4x4** [comment]
Thank you. 
In emulator with Android 7.1 accessibility stay turned on, and app started after reboot : ( 
I add changes proposed by you to code.

**Comment by pharidali** [comment]
Yeah your code was working nicely on emulator but in the real device wasn't .

Let me test your new updated code sir :+1:

**Comment by Gan4x4** [comment]
What models of Xiaomi and Lenovo you had used?

**Comment by pharidali** [comment]
HI ! 
I am using Gionee X1S and Gionee F103 :+1:
